miio/integrations/vacuum/dreame/dreamevacuum_miot.py

```diff
@@ -527,13 +527,22 @@ def set_fan_speed(self, speed: int):
         return self.set_property("cleaning_mode", fanspeed.value)
 
     @command()
-    def fan_speed_presets(self) -> Dict[str, int]:
-        """Return dictionary containing supported fan speeds."""
+    def fan_speed_presets(self) -> FanspeedPresets:
+        """Return available fan speed presets, see VacuumInterface for details."""
```

Comment:
Remove the "please see X for details" commentary all across changes as that's clear from context. Then this is ready to go!

Reply:
There are many methods in each class. How developer can know, which of them are from the interface?
We should either keep complete description for each method or keep reference to the full description.

Reply:
They can always check the inherited classes to check the interface and its docs. Also, the users of the library should develop against the interface, and not the implementation anyway.

Reply:
Yes they can. I the class has 30 method, will they do it for each method? 
And this is not about implementation, it is about usage the method.

Reply:
So I personally see two potential use cases for the interface:
1. You are developing a new integration to add support for a new vacuum device. As you need to implement the interface, you will already want to check the interface to understand what you need to implement.
2. You are a downstream user of the library. If you are targeting to support all vacuum implementations, you will likely want to develop against the interface and not the implementations.

A proper IDE will inform you in both of those cases if you are doing something wrong. Another point is that the docstring is also used for the `miiocli` tool (in `--help` output), which clutters the help. That could be fixed by making the [docstring multiline](https://peps.python.org/pep-0257/#multi-line-docstrings), but I don't feel it's worth the effort (as that reference should also be done in sphinx-format so that the apidoc links are generated correctly).